conf/messages

```diff
@@ -428,17 +428,13 @@ yourAge.TWENTYONEOROVER = 21 or over
 yourAge.error.notCompleted = Select your age
 yourAge.checkYourAnswersLabel = What’s your age?
 
-yourMinimumEarnings.title = Your average weekly earnings
-yourMinimumEarnings.heading = Your average weekly earnings
-yourMinimumEarnings.para1 = We need to know your total weekly pay or profit from self-employment for the next 3 months, before tax and National Insurance are taken off.
-yourMinimumEarnings.para2 = If you’re on maternity, paternity, adoption, or sick leave, it’s how much you earned before you went on leave.
+yourMinimumEarnings.title = On average, will you earn £{0} or more a week?
+yourMinimumEarnings.heading = On average, will you earn £{0} or more a week?
 yourMinimumEarnings.form = On average, will you earn £{0} or more a week?
```

Comment:
The `heading` key is not used anywhere. Same for the partner version. I suggest using `heading` in the views and remove `form` keys for both pages. Or remove `heading` key from `messages` files, but the convention favors the use of `heading` in this scenario (my first suggestion).

Reply:
Please remove the unused message keys: `yourMinimumEarnings.form` and `partnerMinimumEarnings.form`